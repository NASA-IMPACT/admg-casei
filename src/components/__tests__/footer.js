import React from "react"
import renderer from "react-test-renderer"
import Footer from "../footer"

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Footer shortname="NASA | CASEI" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
