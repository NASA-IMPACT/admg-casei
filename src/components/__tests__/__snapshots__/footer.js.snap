// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
.c6.c6 {
  list-style: none;
  margin: 0;
}

.c5.c5 {
  margin: 0 0 1.5rem 0;
  font-weight: bold;
  line-height: 3rem;
}

.c0.c0 {
  margin-top: 5rem;
  background-color: #262a31;
}

.c1.c1 {
  margin: 5rem auto;
  max-width: theme.layout.maxWidth;
  padding: 2rem 5rem;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  display: grid;
  gap: 1rem;
  grid-template-columns: repeat(12,1fr);
}

.c2.c2 {
  grid-column: 1 / span 3;
}

.c3.c3 {
  text-transform: uppercase;
  font-weight: 900;
  font-size: 30px;
  line-height: 2.5rem;
  margin-bottom: 0.5rem;
}

.c4.c4 {
  grid-column: 5 / span 2;
}

.c7.c7 {
  grid-column: 7 / span 2;
}

.c8.c8 {
  grid-column: 9 / span 2;
}

.c9.c9 {
  grid-column: 11 / span 2;
}

.c10.c10 {
  grid-column: 1 / span 12;
  background-color: rgba(255,255,255,0.2);
}

.c11.c11 {
  grid-column: 1 / span 12;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c12.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: bold;
}

.c13.c13 {
  background-image: url("https://www.nasa.gov/sites/default/files/thumbnails/image/nasa-logo-web-rgb.png");
  background-size: cover;
  background-position: center;
  width: 100;
  height: 100;
}

.c14.c14 {
  font-weight: normal;
}

.c15.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  text-transform: uppercase;
  color: #81aefb;
  font-weight: bold;
}

.c16.c16 {
  text-align: center;
}

<footer
  className="c0"
  data-cy="page-footer"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
      >
        NASA | CASEI
      </div>
      <p>
        NASA | CASEI
         is a comprehensive inventory containing information about all airborne and field campaigns as well as aircrafts, instruments, and data products.
      </p>
    </div>
    <div
      className="c4"
    >
      <p
        className="c5"
      >
        Explore
      </p>
      <ul
        className="c6"
      >
        <li>
          <a
            data-cy="footer-campaigns-link"
            href="/explore"
          >
            Campaigns
          </a>
        </li>
        <li>
          <a
            data-cy="footer-platforms-link"
            href="/explore"
            state={
              Object {
                "defaultExploreCategory": "platforms",
              }
            }
          >
            Platforms
          </a>
        </li>
        <li>
          <a
            data-cy="footer-platforms-link"
            href="/explore"
            state={
              Object {
                "defaultExploreCategory": "instruments",
              }
            }
          >
            Instruments
          </a>
        </li>
      </ul>
    </div>
    <div
      className="c7"
    >
      <p
        className="c5"
      >
        Resources
      </p>
      <ul
        className="c6"
      >
        <li>
          <a
            href="/glossary/"
          >
            Glossary
          </a>
        </li>
        <li />
      </ul>
    </div>
    <div
      className="c8"
    >
      <p
        className="c5"
      >
        Quick Links
      </p>
      <ul
        className="c6"
      >
        <li>
          <a
            href="/about/"
          >
            About
          </a>
        </li>
        <li>
          <a
            href="/contact/"
          >
            Contact us
          </a>
        </li>
      </ul>
    </div>
    <div
      className="c9"
    >
      <p
        className="c5"
      >
        Organizations
      </p>
      <ul
        className="c6"
      >
        <li>
          <a
            data-cy="admg-website-link"
            href="https://earthdata.nasa.gov/esds/impact/admg"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#aac9ff",
              }
            }
            target="_blank"
          >
            ADMG Website
          </a>
        </li>
        <li>
          <a
            data-cy="impact-website-link"
            href="https://earthdata.nasa.gov/esds/impact"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#aac9ff",
              }
            }
            target="_blank"
          >
            Impact Website
          </a>
        </li>
      </ul>
    </div>
    <hr
      className="c10"
    />
    <div
      className="c11"
    >
      <div
        className="c12"
      >
        <div
          className="c13"
        />
        <div>
          <div>
            National Aeronautics and Space Administration
          </div>
          <div>
            NASA Official:
             
            <span
              className="c14"
            >
              Rahul Ramachandran
            </span>
          </div>
        </div>
      </div>
      <div
        className="c15"
      />
    </div>
  </div>
  <div
    className="c16"
  >
    Â© 
    2021
    , Built by
     
    <a
      data-cy="devseed-website-link"
      href="https://www.developmentseed.org"
      rel="noopener noreferrer"
      style={
        Object {
          "color": "#aac9ff",
        }
      }
      target="_blank"
    >
      Development Seed
    </a>
    .
  </div>
</footer>
`;
